src/app/scripts/cac/map/cac-map-templates.js

```diff
@@ -320,7 +325,9 @@ CAC.Map.Templates = (function (Handlebars, moment, Utils) {
         });
 
         Handlebars.registerHelper('datetime', function(dateTime) {
-            return new Handlebars.SafeString(new Date(dateTime).toLocaleTimeString());
+            // round to the nearest minute
```

Comment:
I think the issue is that 60+ minutes would get rounded up to 61 minutes instead of rolling over to an hour. I'm going to give [moment-round](https://github.com/WebDevTmas/moment-round) a shot.